webpack.dev.js

```diff
@@ -50,11 +50,6 @@ module.exports = () => ({
     new webpack.WatchIgnorePlugin({
       paths: [/css\.d\.ts$/],
     }),
-    new BundleAnalyzerPlugin({
```

Comment:
Why is this removed?

Reply:
This is not needed in dev at all and it only slows the build process

Reply:
But this wasn't added by us, so I guess that it would be good to talk to someone who can decide if this should stay or not.

Reply:
this was added a long time ago by some client side developer

Reply:
I still think this is a good change as analyzing dev bundle makes no sense because the results are way different than those from the production bundle which is something that really matters